From 188f9efc567bcaef25bde2813cdee7f952f992f9 Mon Sep 17 00:00:00 2001
From: Pat Riehecky <riehecky@fnal.gov>
Date: Mon, 4 Apr 2022 08:38:32 -0500
Subject: [rhel-only PATCH] revert Confine irqbalance to systems where it is useful.

ConditionCPUs only available from systemd(>=242), however the systemd
version for rhel8 is 239. So we need to revert the patch, otherwise we
will encounter the failing:

    $ sudo systemd-analyze verify irqbalance.service
    /usr/lib/systemd/system/irqbalance.service:6: Unknown lvalue
    'ConditionCPUs' in section 'Unit'

This patch revert 188f9efc567bcaef25bde2813cdee7f952f992f9.

---
 misc/irqbalance.service | 1 +
 1 file changed, 1 insertion(+)

diff --git a/misc/irqbalance.service b/misc/irqbalance.service
index fcc29c2..0f79c3e 100644
--- a/misc/irqbalance.service
+++ b/misc/irqbalance.service
@@ -3,7 +3,6 @@ Description=irqbalance daemon
 Documentation=man:irqbalance(1)
 Documentation=https://github.com/Irqbalance/irqbalance
 ConditionVirtualization=!container
-ConditionCPUs=>1
 
 [Service]
 EnvironmentFile=-/usr/lib/irqbalance/defaults.env
-- 
2.33.1

